Source/WebApi.HypermediaExtensions/RESTyard.WebApi.Extensions.csproj

```diff
@@ -28,7 +28,6 @@
     <RepositoryBranch>$(BranchName)</RepositoryBranch>
 
     <!-- Package specific settings -->
-    <PackageId>WebApiHypermediaExtensionsCore</PackageId>
```

Comment:
I think we need this so the NuGet package is build correctly.

Reply:
This is not necessary as the package will have the same name as the project.